kotlinx-coroutines-core/common/src/flow/internal/ChannelFlow.kt

```diff
@@ -126,13 +143,30 @@ internal abstract class ChannelFlowOperator<S, T>(
 internal class ChannelFlowOperatorImpl<T>(
     flow: Flow<T>,
     context: CoroutineContext = EmptyCoroutineContext,
+    val dropWhileBusy: Boolean = false,
```

Comment:
Please, keep the order of `context`, `capacity`, `dropWhileBusy` parameters consistent between constructors, `create`, and `update` functions. Seems logical to add `dropWhileBusy` as the last one.